ml/src/main/java/org/apache/spark/ml/spark/models/svm/SparkSVM.java

```diff
@@ -216,8 +216,7 @@ public void computeImpl() {
                                 svm.run(training, vec2vec(_parms.initialWeights().vecs()));
                 training.unpersist(false);
 
-
-                sc.removeSparkListener(progressBar);
+                sc.listenerBus().removeListener(progressBar);
```

Comment:
removeSparkListener is available only in 2.4 ( maybe 2.3 as well) but not older versions.

`sc.listenerBus().removeListener` is available in 2.4 - 2.1